Rakefile

```diff
@@ -262,6 +263,399 @@ task :xcode => [:dependencies] do
   sh "open #{XCODE_WORKSPACE}"
 end
 
+desc "Install and configure WordPress iOS and it's dependencies - External Contributors"
+namespace :init do
+task :oss => %w[
+  install:xcode:check
+  dependencies
+  install:tools:check_oss
+  install:lint:check
+  credentials:setup
+]
+
+desc "Install and configure WordPress iOS and it's dependencies - a8c Developers"
+task :developer => %w[
+  install:xcode:check
+  dependencies
+  install:tools:check_developer
+  install:lint:check
+  credentials:setup
+  gpg_key:setup
+]
+end
+
+namespace :install do
+  namespace :xcode do
+    task :check => %w[xcode_app:check xcode_select:check]
+
+    #xcode_app namespace checks for the existance of xcode on developer's machine,
+    #checks to make sure that developer is using the correct version per the CI specs
+    #and confirms developer has xcode-select command line tools, if not installs them
+    namespace :xcode_app do
+      #check the existance of xcode, and compare version to CI specs
+      task :check do
+        puts "Checking for system for XCode"
```

Comment:
**Nitpick**, it's called [Xcode](https://developer.apple.com/xcode/) 😆
```suggestion
        puts "Checking for system for Xcode"
```

Reply:
Eek found several of those... fixed